app/client/src/ce/utils/AnalyticsUtil.tsx

```
@@ -50,297 +37,128 @@ export function getParentContextFromURL(location: Location) {
-  static cachedUserId: string;
-  static user?: User = undefined;
-  static blockTrackEvent: boolean | undefined;
   static instanceId?: string = "";
   static blockErrorLogs = false;
+  protected static segmentAnalytics: SegmentSingleton;
+
+  static async initialize(user: User) {
+    SentryUtil.init();
+    SmartlookUtil.init();
+
+    this.segmentAnalytics = SegmentSingleton.getInstance();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)